hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseStore.java

```diff
@@ -215,6 +215,12 @@ private void mutate(Session session, BackendAction item) {
             case ELIMINATE:
                 table.eliminate(session, entry);
                 break;
+            case UPDATE_IF_PRESENT:
+                table.updateIfPresent(session, entry);
+                break;
+            case UPDATE_IF_ABSENT:
```

Comment:
override  updateIfPresent and updateIfAbsent for hbase by `checkAndPut()`, like `checkAndPut(rowkey,family,qualifier,CompareFilter.CompareOp.NOT_EQUAL,null,put)`